cmd/cayley/cayley.go

```diff
@@ -204,6 +207,25 @@ func main() {
 
 		handle.Close()
 
+	case "dump":
+		handle, err = db.Open(cfg)
+		if err != nil {
+			break
+		}
+		if !graph.IsPersistent(cfg.DatabaseType) {
```

Comment:
The irony here is you're about to load a graph to dump a graph.

This is not wrong, however! It's using Cayley as a sort of graph-centered imagemagick-`convert`-esque. :+1: